File: CodeChef/Beginner/03NumberMirror/numbermirror.cpp
#include <iostream>
using namespace std;

class numbermirror {
  int n;

public:
  void takeinput() { cin >> n; }
  void printoutput() { cout << n << endl; }
};

int main() {
  numbermirror num = numbermirror();
  num.takeinput();
  num.printoutput();

  return 0;
}

File: CodeChef/Beginner/03NumberMirror/numbermirror.java
import java.util.Scanner;

class numbermirror {

    public static int TakeInput(Scanner sc) {
        int n = sc.nextInt();
        sc.close();
        return n;
    }

    public static void PrintOutput(int n) {
        System.out.println(n);
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = TakeInput(sc);

        PrintOutput(n);

    }
}